### PR TITLE
Codebase reorganized

### PR DESCRIPTION
**Pull request by Mogball**

### CODE REVIEW

File: cmd/wio/; [hunk 1]
```diff
@@ -5,9 +5,8 @@ import (
     "os"
```
Comment (dhillondeep) [inline cmd/wio/;:5]: Why do we have this file and why is it named ";" ?

File: pkg/util/sys/ioutils.go [hunk 2]
```diff
@@ -127,7 +127,7 @@ func Copy(src string, dst string) error {
     src = filepath.Clean(src)
     dst = filepath.Clean(dst)
     if !Exists(src) {
-        return errors.Stringf("source path [%s] does not exist", src)
+        return errors.New("source path [" + src + "] does not exist")
```
Comment (dhillondeep) [inline pkg/util/sys/ioutils.go:130]: We should use `util.Error` here

Comment (Mogball) [reply pkg/util/sys/ioutils.go:130]: Cyclic imports, was too lazy to find a way around it

**Comment by dhillondeep** [comment]
Should we merge this?

**Comment by Mogball** [comment]
ye I forgot about it

**Pull request merged**